### PR TITLE
Fixes #1380 by simplifying the modal closure mechanism in the 'Manage Features' modal.

### LINKED ISSUE #1380: UI Redundancy with Close Options in Manage Features Modal after Org is Created in OrgList.tsx
**Describe the bug**
In the 'Manage Features' modal, users are presented with two options to close the modal: a clickable 'X' icon and a 'Cancel' button. This presents a UI redundancy that could potentially lead to user confusion. The issue is observable in the OrgList.tsx component on both the `develop` and `main` branches.

**To Reproduce**

1. Navigate to the organization list view on SuperAdmin Screen.
2. Click on 'Create New Organization', fill in the details, and proceed to create the organization.
3. After creation, the ‘Manage Features’ modal will appear.
4. Note the presence of two close options in the top-right corner: an 'X' icon and a 'Cancel' button.

**Expected behavior**
A single, intuitive method should be available for closing the modal to streamline the user experience.

**Actual behavior**
The modal features both an 'X' icon and a 'Cancel' button, offering two similar methods for closing which can be confusing.

**Screenshots**
image

**Additional details**
I'd like to inquire with the maintainers whether the presence of both close options was an intentional design choice. If one of them is to be removed to resolve this UI redundancy, could the maintainers advise which they would prefer? The 'X' icon offers a common modal dismissal pattern, while the 'Cancel' button provides explicit instruction, which might be more accessible. I would be more than happy to work on resolving this. Please assign this issue to me, and I will ensure to make the changes in accordance with the project's design principles.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by harshpreet14**
**What kind of change does this PR introduce?**

This PR addresses issue #1380 by simplifying the modal closure mechanism in the 'Manage Features' modal in OrgList.tsx. 

**Changes Made:**
- Replaced the title div of the Modal with 'Modal Header', aligning it with other modals in the application for consistency.
- Removed the 'Cancel' button and retained the 'X' icon for closing the modal.
- Updated the OrgList.test.tsx file to reflect these changes by removing tests related to the 'CloseOrganizationModal' button.

**Issue Number:**
Fixes #1380 

**Did you add tests for your changes?**
-Verified that - On clicking cross button the modal should close, on clicking outside the modal the modal should close, on pressing escape key on keyboard the modal should close.
- No, there was no need for redundant tests for Modal Header. 
- Executed individual file tests using `npm run test --watchAll=false  /src/screens/OrgList/OrgList.test.tsx`.
- Measured code coverage for changes with `npm run test -- --collectCoverageFrom="src/screens/OrgList/*" /src/screens/OrgList/OrgList.test.tsx`.

**Snapshots/Videos:**
Before- 
Screenshot 2024-01-06 013241
After- 
Screenshot 2024-01-06 184946
OrgList.test.tsx - 
Screenshot 2024-01-06 235928
Code coverage- 
Screenshot 2024-01-07 003019

**If relevant, did you update the documentation?**
NA

**Summary**
Simplifies the modal interface, enhancing the overall user experience and reducing potential confusion.

**Does this PR introduce a breaking change?**

NA

**Other information**
If further tests are required for the specific modal in OrgList.tsx, please let me know, and I'll be more than happy to add them. Thank you for considering my contribution!

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -95,6 +95,9 @@
         "jest-preview": "^0.3.1",
         "jquery": "^3.2.1",
         "postcss-modules": "^6.0.0"
+      },
+      "engines": {
```
Comment (rishav-jha-mech) [inline package-lock.json:99]: Revert any change you make to this file

**Comment by rishav-jha-mech** [comment]
@harshpreet14 Remove the custom CSS from the modal buttons and make them look like the other buttons we are using (Bootstrap buttons)

Also remove all the changes made to Orglist.module.css in this PR
https://github.com/PalisadoesFoundation/talawa-admin/pull/951/files#diff-6d92acde99b887d1558bf07492bfac03d6fdffcab0f3f9b4a52a14df094dbe6c

**Comment by harshpreet14** [comment]
Hi Rishav, I have implemented the changes as suggested. The modifications include updating the buttons to Bootstrap versions, cleaning up unused CSS classes, adding new classes for specific buttons, and implementing an additional test for the 'Go to Plugin Store' button.

**Comment by rishav-jha-mech** [comment]
@harshpreet14 share the updated Screenshot

**Comment by harshpreet14** [comment]
Here's the screenshot @rishav-jha-mech 
image

File: src/screens/OrgList/OrgList.tsx [hunk 2]
```diff
@@ -643,53 +643,40 @@ function orgList(): JSX.Element {
         </Modal>{' '}
         {/* Plugin Notification Modal after Org is Created */}
         <Modal show={dialogModalisOpen} onHide={toggleDialogModal}>
+          <Modal.Header
+            className="bg-primary"
+            closeButton
+            data-testid="modalOrganizationHeader"
```
Comment (beingnoble03) [inline src/screens/OrgList/OrgList.tsx:649]: Rename the `data-testid`. It's already used for other modal above in the same component.

**Review by beingnoble03 — CHANGES_REQUESTED** [review]
@harshpreet14 please address the comment. Rest lgtm.

**Comment by noman2002** [comment]
@harshpreet14 Any update on this ?

**Comment by harshpreet14** [comment]
@noman2002 fixed the redundant test-id. 
Screenshot 2024-01-22 014256

**Comment by noman2002** [comment]
@beingnoble03 Should this be merged ?

**Comment by beingnoble03** [comment]
> @beingnoble03 Should this be merged ?

Yes

**Pull request merged**